### ISSUE TITLE
Percentages exceed 100%

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/cwv-tech-report#8]
It's possible to break the dashboard in such a way that the percentages exceed 100%.

You may see this on the y-axis of the chart or a % column of the data table:

![image](https://user-images.githubusercontent.com/1120896/135953375-909761a3-9888-43aa-8f4b-038fcd32682e.png)

![image](https://user-images.githubusercontent.com/1120896/135953274-cdbbe292-0172-4e61-84f8-eedcf0be166e.png)

The reason why this happens is because some controls are meant to always have exactly one value selected, but Data Studio only has a mechanism in place to prevent 2+ values from being selected and not 0 values. So for example you can deselect the "Client" or "Geo" controls and cause the dashboard to think that all values should be selected at the same time, which causes their percentages to get aggregated together.

I'd love for this to be fixed in Data Studio itself so that the controls can behave more like a radio button, where users are unable to deselect an option that is already selected by default, only change its value. Until then, I'll look for a workaround and leave this issue open to hopefully explain the unexpected behavior.

**Solution:** check the filters/controls on the Technology and Settings pages to ensure that "Client" is always either desktop or mobile and "Geo" has at least one value selected (default "ALL").

**Comment by max-ostapenko** [comment]
@rviscomi FYI Can be fixed by selecting "Single-select" option in Looker Studio.

**Comment by rviscomi** [comment]
Yeah these are single-select fields, so it's impossible to select two ranks or two geos, but that doesn't prevent someone from deselecting the current value, which effectively ends up selecting everything

Anyway, this won't be an issue in the new dashboard so this will become obsolete

**Comment by max-ostapenko** [comment]
Maybe it's about `Allow "Select all"` property.

Does the style config look like this?
<img width="234" alt="Image" src="https://github.com/user-attachments/assets/f39e061e-3f60-4218-bf48-1c49006fc92e" />

**Comment by rviscomi** [comment]
Oh wow yeah that was it! We had it checked for some reason.
